### PR TITLE
feat(alerts): Add tracking to the filter input

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: src/sentry/static/sentry/app/views/settings/incidentRules/ruleForm/index.tsx [hunk 1]
```diff
@@ -286,6 +287,15 @@ class RuleFormContainer extends AsyncComponent<Props, State> {
...
   // searches.
   handleFilterUpdate = (query: string) => {
+    const {organization} = this.props;
+
+    trackAnalyticsEvent({
+      eventKey: 'alert_builder.filter',
+      eventName: 'Alert Builder: Filter',
+      query,
+      organization_id: organization.id,
+    });
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/settings/incidentRules/ruleForm/index.tsx:297]: Just to confirm, this doesn't get called per keystroke?

**Review by billyvg — APPROVED** [review]

File: src/sentry/static/sentry/app/views/settings/incidentRules/ruleForm/index.tsx (hunk 1 above)
Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/views/settings/incidentRules/ruleForm/index.tsx:297]: Good call, I'm pretty sure it's debounced, but now that I think about it, it very well may not be here.

Comment (billyvg) [reply src/sentry/static/sentry/app/views/settings/incidentRules/ruleForm/index.tsx:297]: The comment says it’s only on blur but not sure if accurate

Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/views/settings/incidentRules/ruleForm/index.tsx:297]: Okay, I verified, it is only on blur.

Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/views/settings/incidentRules/ruleForm/index.tsx:297]: Actually it's also debounced on input too. But also blur.

**Pull request merged**